insitu/templates/help.html

```diff
@@ -13,6 +13,12 @@ <h4>
     Can't find a solution? Contact us at
     <a href="mailto:{{ email }}">{{ email }}</a>
   </h4>
+  <h1>Summary</h1>
+  <ul>
+    {% for model, data in models.items %}
+      <li><a href="#{{ model }}">{{ data.nice_name|capfirst }}</a></li>
+    {% endfor %}
+  </ul>
   {% for model, data in models.items %}
```

Comment:
Make sure the title, the contact email text and the list are aligned with the rest of the items.